apps/reference/_storage/intro.mdx

```diff
@@ -0,0 +1,9 @@
+---
+slug: /
+sidebar_position: 1
+sidebar_label: Introduction
```

Comment:
Should we use `Introduction` or the name of the reference (e.g., https://reference-docs-git-chore-v2docs-supabase.vercel.app/cli)?

![Screen Shot 2022-08-02 at 4 05 49 PM](https://user-images.githubusercontent.com/7026076/182489119-9b149dc7-d246-4e43-8fe1-bf2c26387519.png)